### PR TITLE
Fix errors not updated for other module

### PR DESCRIPTION
**Pull request by br1anchen**
**What kind of change does this PR introduce?**
Bug fix for #538: 
Update errors info when module changed

**What is the current behavior?**
Error changes caused by current editor module will not be notified by other module

**What is the new behavior?**
When module changed, it will try to update with current erros

**Checklist**:

- ~~[ ] Documentation~~
- ~~[ ] Tests~~
- [x] Ready to be merged 
- [x] Added myself to contributors table 

Current implementation is to update errors for new module when module changed. I think it is not necessary to update all errors for all related module except current edit module since codesandbox.io is always editing (single?) module.
It is efficient enough to update errors info when module changed.

### CODE REVIEW

**Comment by CompuIves** [comment]
Oh this is really great! Thank you very much @br1anchen!! Could you fix the remaining lint errors? Then it's ready for merge.

File: packages/app/src/app/pages/Sandbox/Editor/Content/index.js [hunk 1]
```diff
@@ -185,15 +185,16 @@ class EditorPreview extends React.Component<Props, State> {
     );
     const disposeModuleHandler = reaction(
       () => [store.editor.currentModule, store.editor.currentModule.code],
-      () => {
+      ([newModule, _]) => {
```
Comment (CompuIves) [inline packages/app/src/app/pages/Sandbox/Editor/Content/index.js]: I think you can just change this line to:

`([newModule]) => {`

Comment (br1anchen) [reply packages/app/src/app/pages/Sandbox/Editor/Content/index.js]: True, thanks for your input.

**Comment by br1anchen** [comment]
@CompuIves by the way, is there any reason we are using forked version `monaco-editor` here? For package stability?

**Comment by CompuIves** [comment]
Great! Thanks for this PR 😄 

> is there any reason we are using forked version monaco-editor here? For package stability?

I use a version fo monaco-editor that is a bit more integrated with CodeSandbox, for example, you can open files by CMD+Clicking on the path in the code.

**Pull request merged**